index.html

```diff
@@ -4,7 +4,7 @@
   <script type="module" src="sirius-header.js"></script>
   <link rel="stylesheet" href="css/all.css">
 </head>
-<body class="govuk-template__body">
+<body class="govuk-template__body app-body-class">
```

Comment:
What is `app-body-class`? It doesn't appear to be doing anything.

Reply:
Yup sorry I'll get rid of this.